comid/tdx-profile/mval_extensions.go

```diff
@@ -0,0 +1,55 @@
+package tdx
+
+import (
+	"time"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/corim/corim"
+	"github.com/veraison/corim/extensions"
+	"github.com/veraison/eat"
+)
+
+// MValExtensions contains the Intel TDX profile extensions which can appear in
+// both Reference Values and Endorsed Values
+type MValExtensions struct {
+	TeeTcbDate     *time.Time      `cbor:"-72,keyasint,omitempty" json:"tcbdate,omitempty"`
+	TeeISVSVN      *TeeSVN         `cbor:"-73,keyasint,omitempty" json:"isvsvn,omitempty"`
+	TeeInstanceID  *TeeInstanceID  `cbor:"-77,keyasint,omitempty" json:"instanceid,omitempty"`
+	TeePCEID       *TeePCEID       `cbor:"-80,keyasint,omitempty" json:"pceid,omitempty"`
+	TeeMiscSelect  *TeeMiscSelect  `cbor:"-81,keyasint,omitempty" json:"miscselect,omitempty"`
+	TeeAttributes  *TeeAttributes  `cbor:"-82,keyasint,omitempty" json:"attributes,omitempty"`
+	TeeMrTee       *TeeDigest      `cbor:"-83,keyasint,omitempty" json:"mrtee,omitempty"`
+	TeeMrSigner    *TeeDigest      `cbor:"-84,keyasint,omitempty" json:"mrsigner,omitempty"`
+	TeeISVProdID   *TeeISVProdID   `cbor:"-85,keyasint,omitempty" json:"isvprodid,omitempty"`
+	TeeTcbEvalNum  *TeeTcbEvalNum  `cbor:"-86,keyasint,omitempty" json:"tcbevalnum,omitempty"`
+	TeeTcbStatus   *TeeTcbStatus   `cbor:"-88,keyasint,omitempty" json:"tcbstatus,omitempty"`
+	TeeAdvisoryIDs *TeeAdvisoryIDs `cbor:"-89,keyasint,omitempty" json:"advisoryids,omitempty"`
+	TeeEpoch       *time.Time      `cbor:"-90, keyasint,omitempty" json:"epoch,omitempty"`
+
+	TeeCryptoKeys *comid.CryptoKeys `cbor:"-91, keyasint,omitempty" json:"teecryptokeys,omitempty"`
+	TeeTCBCompSvn *TeeTcbCompSvn    `cbor:"-125, keyasint,omitempty" json:"tcbcompsvn,omitempty"`
+}
+
+// Registering the profile inside init() in the same file where it is defined
+// ensures that the profile will always be available, and you don't need to
+// remember to register it at the time you want to use it. The only potential
+// danger with that is if the your profile ID clashes with another profile,
+// which should not happen if it a registered PEN or a URL containing a domain
+// that you own.
+
+func init() {
+	profileID, err := eat.NewProfile("http://intel.com/tdx-profile")
```

Comment:
It's not versioned? And it's not https? And isn't CoRIM specific? I'd expect there to be an EAT profile for TDX as well that would clash.

Ned needs to add a /v1 or and ensure it's https-protected.

Reply:
Need to discuss this with Intel CoRIM Profile authors. Now the Profile in this PR is according to the profile documented in the Intel profile document!